packages/sdks/src/functions/get-processed-block.ts

```diff
@@ -109,18 +107,17 @@ export function getProcessedBlock({
   BuilderContextInterface,
   'localState' | 'context' | 'rootState' | 'rootSetState'
 >): BuilderBlock {
-  let transformedBlock = resolveLocalizedValues(
-    block,
-    rootState.locale as string | undefined
-  );
-
-  transformedBlock = transformBlock(transformedBlock);
-
-  return evaluateBindings({
+  let transformedBlock = transformBlock(block);
+  transformedBlock = evaluateBindings({
     block: transformedBlock,
     localState,
     rootState,
     rootSetState,
     context,
   });
+  const localizedBlock = resolveLocalizedValues(
+    transformedBlock,
+    rootState.locale as string | undefined
+  );
+  return localizedBlock;
```

Comment:
minor thing but could we just use `transformedBlock` variable instead of creating a new variable?
```suggestion
  transformedBlock = resolveLocalizedValues(
    transformedBlock,
    rootState.locale as string | undefined
  );
  return transformedBlock;
```